azure-pipelines.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       displayName: 'Use Python 3.6'
       inputs:
         versionSpec: 3.6
```

Comment:
We'd better update python version so that we can install the latest pylint and flake8.